tests/compute.py

```diff
@@ -435,3 +435,100 @@ def compute_image_3d_load_store(cfg: SceneCfg, show_dbg_points=False):
         group.add_children(get_debug_points(cfg, cuepoints))
 
     return group
+
+
+_IMAGE_CUBE_STORE_COMPUTE = """
+void main()
+{
+    ivec3 pos = ivec3(gl_LocalInvocationID.xyz);
+    vec4 color = vec4(0.0);
+    if (pos.z == 0) // right
+        color = vec4(1.0, 0.0, 0.0, 1.0);
+    else if (pos.z == 1) // left
+        color = vec4(0.0, 1.0, 0.0, 1.0);
+    else if (pos.z == 2) // top
+        color = vec4(0.0, 0.0, 1.0, 1.0);
+    else if (pos.z == 3) // bottom
+        color = vec4(1.0, 1.0, 0.0, 1.0);
+    else if (pos.z == 4) // back
+        color = vec4(0.0, 1.0, 1.0, 1.0);
+    else if (pos.z == 5) // front
+        color = vec4(1.0, 0.0, 1.0, 1.0);
+    imageStore(texture, pos, color);
+    imageStore(texture, pos, color);
+    imageStore(texture, pos, color);
```

Comment:
Is this on purpose, 3 times the same instruction? 🤔

Reply:
A leftover from a previous iteration, sorry.